src/FSLibrary/MissionMaxTPSMixed.fs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionMaxTPSMixed
+
+// This module provides a max TPS test with a blend of classic and Soroban load.
+// It uses the MaxTPSTest module to perform a binary search for the max TPS.
+
+open MaxTPSTest
+open StellarMissionContext
+open StellarCoreHTTP
+
+// If `list` is empty, return `value`. Otherwise, return `list`.
+let private defaultListValue value list =
+    match list with
+    | [] -> value
+    | _ -> list
+
+let maxTPSMixed (baseContext: MissionContext) =
+    let context =
+        { baseContext with
+              coreResources = SimulatePubnetTier1PerfResources
+              installNetworkDelay = Some(baseContext.installNetworkDelay |> Option.defaultValue true)
+              // Simulate apply duration using same distribution as
+              // `MissionSimulatePubnet`
+              simulateApplyDuration = Some(baseContext.simulateApplyDuration |> Option.defaultValue (seq { 0 }))
```

Comment:
I believed you mentioned this was going to be removed?

Reply:
Ah, I meant that I had set simulated apply duration to `0` to match the classic TPS test. This has some other impacts (such as using the in-memory SQL database) that I think we want, right? But given what we've talked about in https://github.com/stellar/stellar-core/issues/4365 maybe I should remove it entirely. What do you think?

Also, the comment on this line was outdated so I removed it.

Reply:
Yeah I think we should remove it. It's a bit confusing to have the flag on, given that it doesn't behave as expected. (Let's keep it in the classic test though, since it works correctly there)

Reply:
Makes sense. I removed it in 324bdd179e5bcadaf007d217dc231c6a3474b6b7